.github/workflows/tests-nightly.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       run: |
         echo "Running unit tests for server: ${{ matrix.testServer }}"
         mkdir -p ~/logs-unit
-        ./gradlew runTestsWithTag -PtestTag=unit --scan -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-unit -PquietConsole
+        ./gradlew runTestsWithTag -PtestTag=unit -Pplatform=GCP -Pserver=${{ matrix.testServer }} -PcontextDir=$HOME/logs-unit -PquietConsole --scan
```

Comment:
added to integration tests as well in https://github.com/DataBiosphere/terra-cli/pull/479